test/sax/sax_test.rb

```diff
@@ -263,8 +261,7 @@ def test_sax_nested1
                    [:start_element, :grandchild],
                    [:end_element, :grandchild],
                    [:end_element, :child],
-                   [:end_element, :top],
-                  ])
+                   [:end_element, :top],])
```

Comment:
Please keep the original.

Reply:
There are a few options for this rule https://docs.rubocop.org/rubocop/cops_layout.html#layoutmultilinearraybracelayout
I think if the opening braket at the start was on a newline it would keep it, or the explicit `EnforcedStyle: new_line`

Reply:
Id be happy with it is the original tail looks is kept by putting the opening on a new line.